test/gpu_tests.jl

```diff
@@ -19,29 +21,29 @@ import CloudMicrophysics.Common as CO
 import CloudMicrophysics.CommonTypes as CMT
 import CloudMicrophysics.Microphysics0M as CM0
 import CloudMicrophysics.Microphysics1M as CM1
-import CloudMicrophysics.Nucleation as HN
-import CloudMicrophysics.Parameters.H2S04NucleationParameters
-import CloudMicrophysics.Parameters.MixedNucleationParameters
-import CloudMicrophysics.Parameters.OrganicNucleationParameters
+import CloudMicrophysics.Nucleation as MN
 import CloudMicrophysics.P3Scheme as P3
 
 const kaolinite = CMT.KaoliniteType()
 const illite = CMT.IlliteType()
 
-@info "GPU Tests"
+const work_groups = (1,)
 
 if ClimaComms.device() isa ClimaComms.CUDADevice
-    import CUDA
-    ArrayType = CUDA.CuArray
+    using CUDA
+    using CUDA.CUDAKernels
+    const backend = CUDABackend()
     CUDA.allowscalar(false)
-    device(::Type{T}) where {T <: CUDA.CuArray} = CK.CUDADevice()
+    const ArrayType = CuArray
 else
-    ArrayType = Array
```

Comment:
Since we now only run the tests on the GPU, would it make sense to throw an error here? This way we will know that our tests is actually running on the GPU and not silently on the CPU?

(I would maybe just comment out those two lines, so that it's easy to switch to PU for debugging?)